TESTING_GUIDE.md

```
@@ -1,33 +1,57 @@
+- [Test Best Practices](#test-best-practices)
+- [Example Test Session](#example-test-session)
+- [Continuous Integration](#continuous-integration)
+- [Test Documentation](#test-documentation)
+- [Support](#support)
+
+---
+
 ## Quick Start
 
 To run all tests:
 ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)